File: packages/expo-camera/android/src/main/java/expo/modules/camera/tasks/FaceDetectorAsyncTaskDelegate.java
package expo.modules.camera.tasks;

import android.os.Bundle;

import java.util.List;

import expo.modules.interfaces.facedetector.FaceDetectorInterface;

public interface FaceDetectorAsyncTaskDelegate {
  void onFacesDetected(List<Bundle> faces);
  void onFaceDetectionError(FaceDetectorInterface faceDetector);
  void onFaceDetectingTaskCompleted();
}

File: packages/expo-camera/android/src/main/java/expo/modules/camera/tasks/FaceDetectorAsyncTaskDelegate.kt
package expo.modules.camera.tasks

import android.os.Bundle
import expo.modules.interfaces.facedetector.FaceDetectorInterface

interface FaceDetectorAsyncTaskDelegate {
  fun onFacesDetected(faces: List<Bundle>)
  fun onFaceDetectionError(faceDetector: FaceDetectorInterface)
  fun onFaceDetectingTaskCompleted()
}
